src/SFA.DAS.AssessorService.Application.Api/Validators/ExternalApi/Certificates/GetBatchCertificateValidator.cs

```diff
@@ -18,20 +18,7 @@ public GetBatchCertificateRequestValidator(IStringLocalizer<GetBatchCertificateR
             RuleFor(m => m.UkPrn).InclusiveBetween(10000000, 99999999).WithMessage("The UKPRN should contain exactly 8 numbers");
 
             RuleFor(m => m.FamilyName).NotEmpty().WithMessage("Provide apprentice family name");
-            RuleFor(m => m.StandardCode).GreaterThan(0).WithMessage("Provide a valid Standard").DependentRules(() =>
-            {
-                RuleFor(m => m).CustomAsync(async (m, context, cancellation) =>
-                {
-                    if (!string.IsNullOrEmpty(m.StandardReference))
-                    {
-                        var collatedStandard = await standardService.GetStandard(m.StandardReference);
-                        if (m.StandardCode != collatedStandard?.StandardId)
-                        {
-                            context.AddFailure("StandardReference and StandardCode must be for the same Standard");
-                        }
-                    }
-                });
-            });
+            RuleFor(m => m.StandardCode).GreaterThan(0).WithMessage("Provide a valid Standard");
```

Comment:
I'm debating this validation line. They provide a lars code or ifate reference which we use to get the latest standard. Then we populate both the StandardCode and StandardReference on the request. So in theory either they will both be null or both be set. 

I guess this is a catch all so that if it's not set, a valid id wasn't supplied.